src/lib/account-profile/account-profile-menu/account-profile-menu.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatIconModule } from '@angular/material/icon';
+import { MatListModule } from '@angular/material/list';
+import { CovalentMenuModule } from '@covalent/core/menu';
+
+import { VantageAccountProfileMenuComponent } from './account-profile-menu.component';
+
+describe('VantageAccountProfileMenuComponent', () => {
+  let component: VantageAccountProfileMenuComponent;
+  let fixture: ComponentFixture<VantageAccountProfileMenuComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [VantageAccountProfileMenuComponent],
+      imports: [MatIconModule, MatListModule, CovalentMenuModule],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(VantageAccountProfileMenuComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    component.email = 'daffy.duck@teradata.com';
+    component.name = 'Daffy Duck';
+    fixture.detectChanges();
+
+    expect(component).toBeTruthy();
+  });
+
+  it('should block out click event on header click', () => {
+    const mouseEvent: MouseEvent = new MouseEvent("click",{bubbles: true, cancelable: true});
```

Comment:
How about we test whether the menu stays open or not. Will require some changes to the fixture.